src/app/features/psbt-signer/components/psbt-request-details/components/psbt-inputs-and-outputs/psbt-inputs-and-outputs.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+
+import { PsbtInput } from '@app/features/psbt-signer/hooks/use-parsed-inputs';
+import { PsbtOutput } from '@app/features/psbt-signer/hooks/use-parsed-outputs';
+
+import { PsbtRequestDetailsSectionHeader } from '../psbt-request-details-section-header';
+import { PsbtRequestDetailsSectionLayout } from '../psbt-request-details-section.layout';
+import { PsbtInputList } from './components/psbt-input-list/psbt-input-list';
+import { PsbtOutputList } from './components/psbt-output-list/psbt-output-list';
+
+interface PsbtInputsAndOutputsProps {
+  addressNativeSegwit: string;
+  addressTaproot: string;
+  inputs: PsbtInput[];
+  outputs: PsbtOutput[];
+}
+
+export function PsbtInputsAndOutputs({
+  addressNativeSegwit,
+  addressTaproot,
+  outputs,
+  inputs,
+}: PsbtInputsAndOutputsProps) {
+  const [showDetails, setShowDetails] = useState(false);
+
+  if (!inputs.length && !outputs.length) return null;
```

Comment:
don't we expect both inputs and outputs here? shouldn't it be 
```
if (!inputs.length || !outputs.length)
```
?

Reply:
Yep, good point, it prob should be on `or`. Thanks.